src/Sql/Sql.Sdk/Generated/BackupShortTermRetentionPoliciesOperations.cs

```diff
@@ -94,6 +94,21 @@ public async Task<AzureOperationResponse<BackupShortTermRetentionPolicy>> GetWit
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "serverName");
             }
+            if (serverName != null)
+            {
+                if (serverName.Length > 63)
+                {
+                    throw new ValidationException(ValidationRules.MaxLength, "serverName", 63);
+                }
+                if (serverName.Length < 3)
+                {
+                    throw new ValidationException(ValidationRules.MinLength, "serverName", 3);
+                }
+                if (!System.Text.RegularExpressions.Regex.IsMatch(serverName, "^[a-z][a-z0-9]*$"))
```

Comment:
server name can accept hyphens as well (`-`)

Reply:
@vidai-msft where do these validations originate from?

Reply:
Used a new commit id where the serverName validation was reverted.

Reply:
serverName validation was reverted.